### DISCUSSION TITLE
Add multiple Recipes to shopping List

### DISCUSSION THREAD

**Discussion by josuabarron** [mealie-recipes/mealie#3440] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

I plan my week ahead and then i have to export every recepie to the shoping List.

### Please provide a concise description of the feature that would resolve your issue.

It would be great to have a button to export a whole week to the shopping list and to see what i am adding and check things that i still have at home.

### Please consider and list out some caveats or tradeoffs made in your design decision

I dont thing there are any tradeoffs.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by michael-genson** [comment]
What version are you on? This has been a feature for a few versions now on the meal planner page

**Comment by josuabarron** [comment]
Oh yes, I am currently on an older version due to umbrel. Updating to a new VM soon. I also tried the demo online.

But what I was missing is a button to add multiple meals to the List at the same time. That would be amazing.

Thanks for the help tho :)
